packages/file/src/index.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * This file is the entry point to {@link @sketch-hq/sketch-file}.
+ */
+
+import path from 'path'
+import StreamZip from 'node-stream-zip'
+import FileFormat from '../../file-format-ts'
+import Zip from 'adm-zip'
+
+/**
+ * Given a path to a Sketch file on the file system, this function unzips the
+ * JSON entries and parses them out into a SketchFile object.
+ */
+const fromFile = async (filepath: string): Promise<SketchFile> => {
```

Comment:
Would it be nicer to have `fromFile` and `toFile` functions in their own files? Might be more scalable that way if new functions come in in future. No strong opinions either way at this stage though ...